api-spec/protobuf/operator.proto

```diff
@@ -151,6 +151,9 @@ service Operator {
 
   // Returns the list of all withdrawals made for the given account.
   rpc ListWithdrawals(ListWithdrawalsRequest) returns(ListWithdrawalsReply){}
+
+  // Returns info about volume and collected fees for specific time range
+  rpc GetMarketReport(GetMarketReportRequest) returns(GetMarketReportsReply){}
```

Comment:
I think with this rpc we can get rid of the GetMarketCollectedFees one?
@tiero @sekulicd what do you think?

Reply:
The idea is to returns the paginated list of collected fee for a trade in GetMarketCollectedFees